### ISSUE TITLE
[Feature request] Be able to access the lastname prefix + lastname separately 

### ISSUE THREAD

**Issue by divinenanny** [theiconic/name-parser#10]
I really like how the name-parser can recognise the lastname prefix correctly. Now I would like to be able to get this prefix and the lastname separately. 
This could be a setting (maybe related to the language setting, where in Dutch the lastname prefix is a separate part of the name (and for example sorting goes by last name, without the prefix).)

Example:

Frank van Delft (Dutch name), is recognised as:

```php
TheIconic\NameParser\Name Object
(
    [parts:protected] => Array
        (
            [0] => TheIconic\NameParser\Part\Firstname Object
                (
                    [value:protected] => Frank
                )

            [1] => TheIconic\NameParser\Part\LastnamePrefix Object
                (
                    [normalized:protected] => van
                    [value:protected] => Van
                )

            [2] => TheIconic\NameParser\Part\Lastname Object
                (
                    [value:protected] => Delft
                )

        )

)
```

Doing:
```php
echo $name->getFirstname() . "\n";
echo $name->getLastnamePrefix() . "\n";
echo $name->getLastname() . "\n";
```

Gives:
```
Frank
van
Delft
```

**Comment by wyrfel** [comment]
Hi @divinenanny. Sounds like a reasonable change. Would you be game to create a PR for this yourself?

**Comment by wyrfel** [comment]
Actually just implemented it in #13.
